src/lib.rs

```diff
@@ -87,6 +89,9 @@ impl AcarsVdlm2Message {
                 vdlm2.clear_station_name(),
             AcarsVdlm2Message::AcarsMessage(acars) =>
                 acars.clear_station_name(),
+            AcarsVdlm2Message::HfdlMessage(hfdl) => {
```

Comment:
This can simply be written as:
`AcarsVdlm2Message::HfdlMessage(hfdl) => hfdl.clear_station_name(),` or

```
AcarsVdlm2Message::HfdlMessage(hfdl) =>
    hfdl.clear_station_name(),
```

If you want a shorter line length.

Only need to add `{ ... }` if there's more than one action on the match arm.

Same would go for the other uses below